medusa/subtitle_providers/legendastv.py

```diff
@@ -436,7 +445,7 @@ def query(self, language, title, season=None, episode=None, year=None):
                 for r in releases:
                     subtitle = LegendasTVSubtitle(language, t['type'], t['title'], t.get('year'), t.get('imdb_id'),
                                                   t.get('season'), a, r)
-                    logger.debug('Found subtitle %r', subtitle)
+                    logger.info('Found subtitle %r', subtitle)
```

Comment:
This is the only thing. That should still be debug in subliminal world. 
All the rest is great 👍